src/TgSharp.Core/Session.cs

```diff
@@ -107,11 +159,19 @@ public class Session
         public long LastMessageId { get; set; }
         public int SessionExpires { get; set; }
         public TLUser TLUser { get; set; }
-        private Random random;
+        private readonly Random random = new Random();
+
+        [JsonIgnore]
+        public ISessionStore Store;
```

Comment:
now, after my refactoing commit is there, I guess removing this field is much easier now